src/service/viewport-impl.js

```diff
@@ -453,6 +453,22 @@ export class Viewport {
   }
 
   /*
+   * Instruct the viewport to enter sidebar mode.
+   */
+  enterSidebarMode() {
```

Comment:
@camelburrito We cannot reuse `enterLightboxMode` because sidebar doesn't need `hideFixedLayer` and `binding_.updateLightboxMode`.
@aghassemi Maybe I can rename it to `enterOverlayMode` ? In case some other extension want to reuse these two methods.